doc/api_methods/SubscribeToLists.md

```diff
@@ -0,0 +1,27 @@
+[back to list](../Readme.md)
+
+# Subscribe to Lists
+
+## `array subscribeToLists(string $subscriber_id, array $list_ids [, array $options = []])`
+
+This method allows adding an existing subscriber into lists and handles confirmation email and admin notification email sending.
+
+*A confirmation email* is an email which is sent to a subscriber so that he can confirm his subscription. It is sent only if sign-up confirmation is enabled in MailPoet settings and subscriber has not received any confirmation email yet.
+*A welcome email* is an automatic email which is sent to a new subscriber. This email is scheduled only if sign-up confirmation is disabled and some welcome email is configured for some of given lists.
+
+It returns a subscriber. See [Get Subscriber](GetSubscriber.md) for a subscriber data structure.
+
+## Arguments
+### string `$subscriber_id` (required)
+An id or email of an existing subscriber. An `\Exception` is thrown when an id or email doesn't match any subscriber.
+
+### array `$list_ids` (required)
+An array of list ids. An `\Exception` is thrown if any of list ids is invalid.
```

Comment:
What happens if someone passes a list of ids and one of them doesn't exist? Does it subscribe to some of them? or none of them?